src/coreclr/nativeaot/BuildIntegration/Microsoft.NETCore.Native.Windows.targets

```diff
@@ -23,7 +23,7 @@ The .NET Foundation licenses this file to you under the MIT license.
     <FullRuntimeName>Runtime.WorkstationGC</FullRuntimeName>
     <FullRuntimeName Condition="'$(ServerGarbageCollection)' == 'true' or '$(IlcLinkServerGC)' == 'true' or '$(ControlFlowGuard)' == 'Guard'">Runtime.ServerGC</FullRuntimeName>
     <BootstrapperName>bootstrapper</BootstrapperName>
-    <BootstrapperName Condition="'$(NativeLib)' != '' or '$(CustomNativeMain)' == 'true'">bootstrapperdll</BootstrapperName>
+    <BootstrapperName Condition="'$(NativeLib)' != ''">bootstrapperdll</BootstrapperName>
```

Comment:
@jkotas should we also keep this condition like in the Unix file? As in, is the bootstrapper .dll always going to be used when a custom main is used, regardless of whether the native binary being produced is actually a .dll or an .exe?

Reply:
Yes, the Windows and Unix should be in sync.